liquibase-snowflake/src/main/java/liquibase/sqlgenerator/core/GetViewDefinitionGeneratorSnowflake.java

```diff
@@ -0,0 +1,35 @@
+package liquibase.sqlgenerator.core;
+
+import liquibase.CatalogAndSchema;
+import liquibase.database.Database;
+import liquibase.database.core.SnowflakeDatabase;
+import liquibase.sql.Sql;
+import liquibase.sql.UnparsedSql;
+import liquibase.sqlgenerator.SqlGeneratorChain;
+import liquibase.statement.core.GetViewDefinitionStatement;
+
```

Comment:
@LonwoLonwo Can you please add javadoc comments to public class?

Reply:
Sure. Added.

I didn't find any specific pattern for Javadoc for classes, So I hope my variant is standard.